### PR TITLE
[docs-only] Cleanup docs

### PR DESCRIPTION
**Pull request by micbar**
## Description

- [x] Clean up the docs in the remaining extension folders
- [x] Make sure that every Makefile has a `config-docs-generate` target
- [x] Bring webdav docs to the new format
- [x] Add config docs generate to the pipeline

### CODE REVIEW

**Comment by micbar** [comment]
@kulmann @pmaier1 See the power of the new docs pipeline. Done in 1:16 min.

File: accounts/Makefile [hunk 1]
```diff
@@ -135,24 +135,9 @@ release-check:
...
-
-.PHONY: docs-build
-docs-build:
-	cd $(HUGO); hugo
-
-.PHONY: docs
-docs: docs-copy docs-build
+.PHONY: config-docs-generate
+config-docs-generate:
+#	go run github.com/owncloud/flaex >| ../docs/extensions/$(NAME)/configuration.md
```
Comment (kulmann) [inline accounts/Makefile:140]: Can you write a ticket that we want to enable flaex later on for the extensions that don't have it yet?

**Review by kulmann — APPROVED** [review]
The services/ports table on the reva page is broken - maybe can be handled in a different PR. Looks good otherwise 👍

**Review by kulmann — CHANGES_REQUESTED** [review]
just seeing now: the text and logo on the start page of the docs are broken.

**Comment by micbar** [comment]
did you do a local build?

**Comment by kulmann** [comment]
Yep, it failed on the local build

**Comment by micbar** [comment]
@kulmann fixed it
Screenshot_2020-10-01 Public upload Flow ownCloud
Screenshot_2020-10-01 Documentation ownCloud

**Review by kulmann — APPROVED** [review]
🚀

File: accounts/Makefile (hunk 1 above)
Comment (micbar) [reply accounts/Makefile:140]: 👍 done https://jira.owncloud.com/browse/OCIS-655

**Pull request merged**